android/Staccato_AN/app/src/main/java/com/on/staccato/presentation/staccatocreation/StaccatoCreationActivity.kt

```diff
@@ -296,6 +296,7 @@ class StaccatoCreationActivity :
             viewModel.fetchPhotosUrlsByUris(this)
         }
         viewModel.currentPhotos.observe(this) { photos ->
+            photoAttachFragment.setCurrentImageCount(StaccatoCreationViewModel.MAX_PHOTO_NUMBER - photos.size)
             photoAttachAdapter.submitList(
                 listOf(AttachedPhotoUiModel.addPhotoButton).plus(photos.attachedPhotos),
```

Comment:
```suggestion
                listOf(AttachedPhotoUiModel.addPhotoButton, *photos.attachedPhotos.toTypedArray()),
```

두 개의 list를 합치기 보다는 가변인자를 함수의 인자로 넘길 수 있는 listOf의 특성과 스프레드 연산자를 활용하는 건 어떠신가요?